src/codegen/riscv64/assembler-riscv64-inl.h

```diff
@@ -179,6 +185,17 @@ Address RelocInfo::target_internal_reference_address() {
   return pc_;
 }
 
+Handle<Code> Assembler::relative_code_target_object_handle_at(
+    Address pc) const {
+  Instr instr1 = Assembler::instr_at(pc);
+  Instr instr2 = Assembler::instr_at(pc + kInstrSize);
+  DCHECK(IsAuipc(instr1));
+  DCHECK(IsJalr(instr2));
+  // Interpret 2 instructions generated by li (lui/ori).
```

Comment:
I don't understand this comment. Aren't the two instructions an `auipc` and a `jalr`?

Reply:
done